runtime/src/server/middleware/index.ts

```diff
@@ -17,6 +17,9 @@ export default function middleware(opts: {
 	return compose_handlers(ignore, [
 		(req: Req, res: Res, next: () => void) => {
 			if (req.baseUrl === undefined) {
+				if (!req.originalUrl) {
+					req.originalUrl = req.url
+				}
 				let { originalUrl } = req;
```

Comment:
@Rich-Harris was proposing the same change in https://github.com/sveltejs/sapper/pull/682/files#r432806606. This seems pretty reasonable to me. I think his way of setting it was a bit more succinct though. What do you think about updating it to that?

Reply:
Yes, this was only a proposition to solve #923, it's clearer and could be safer.
Should I close the pull or update it to match his changes?